Extension/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.17.7: July 19, 2018
+## Version 0.17.7: July 22, 2018
```

Comment:
0.17.7 already shipped -- can you add a 0.17.8, with some date such as August 16?